tsdb/engine/tsm1/file_store.go

```diff
@@ -485,12 +505,43 @@ func (f *FileStore) Replace(oldFiles, newFiles []string) error {
 	}
 
 	// We need to prune our set of active files now
-	var active []TSMFile
+	var active, inuse []TSMFile
 	for _, file := range updated {
 		keep := true
 		for _, remove := range oldFiles {
 			if remove == file.Path() {
 				keep = false
+
+				// If queries running against this file, then we need to move it out of the
```

Comment:
"If queries running against this file" -> "If there are queries running against this file"